Cargo.toml

```diff
@@ -33,6 +33,7 @@ rocksdb = { version = "0.14", default-features = false, features = ["snappy"], o
 cc = { version = ">=1.0.64", optional = true }
 socks = { version = "0.3", optional = true }
 lazy_static = { version = "1.4", optional = true }
+hwi = { path = "../rust-hwi", optional = true }
```

Comment:
This should be a `version =` kinda path

Reply:
Yep. I specified a local path since I know this won't work on the published version of hwi (and I did the testing on a local crate so just left it like that 😅). I opened a PR on rust-hwi for this.

Reply:
Ah ok, I can merge and tag a new minor version then

Reply:
Btw, while I wait before tagging the new version, you can use

`hwi = { git = "https://github.com/bitcoindevkit/rust-hwi.git" }`

so that you can run the CI :) Also, you can specify a branch, in case you have more fixes to rust-hwi but can't wait for my (slow :snail:) merges

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories